core-web/libs/sdk/react/package.json

```diff
@@ -4,7 +4,7 @@
   "peerDependencies": {
     "react": ">=18",
     "react-dom": ">=18",
-    "@dotcms/client": "0.0.1-alpha.38",
+    "@dotcms/client": "latest",
```

Comment:
WTF?

Reply:
yeah, we forgot to change it, all of the libs are using `latest` now